config/default.json

```diff
@@ -91,6 +91,7 @@
     "maxNetworkTaskAttempts": 1,
     "maxCartBypassAttempts": 1
   },
+  "appName": "Vuestore",
```

Comment:
How about we change the appName to something more meaningful like storeName? Also, the docs are missing. We're trying to keep every config entry documented.

Reply:
- Okay, I'll take care of it.

Reply:
it'll be better like moving it to `config.seo.defaultTitle` OK?

Reply:
@pkarw Yeah, It's perfect.
So, Do you need this title should base on stores?

Reply:
yeah it should be set in `config.storeViews.de.seo.defaultTitle` as well - I mean per each storeview

Reply:
Perfect.
Let me do this way.